server/superdesk/publish/odbc_test.py

```diff
@@ -15,7 +17,7 @@
 
 
 class ODBCTests(TestCase):
-    subscribers = [{"_id": "1", "name": "Test", "can_send_takes_packages": False, "media_type": "media",
+    subscribers = [{"_id": "1", "name": "Test", "subscriber_type": SUBSCRIBER_TYPES.WIRE, "media_type": "media",
```

Comment:
You mean to say that Digital/Internet means non-media client and rest are media clients.
@amagdas I think it is better to get confirmation from POs.